README.md

```diff
@@ -3,21 +3,16 @@
 
 Golang bindings for Mozilla's [DeepSpeech](https://github.com/mozilla/DeepSpeech) speech-to-text library.
 
-As of now, `astideepspeech` is only compatible with version `v0.1.1` of `DeepSpeech`.
+As of now, `astideepspeech` is only compatible with version `v0.4.0` of `DeepSpeech`.
 
 # Installation
 ## Install DeepSpeech
 
-- get the code by following instructions in DeepSpeech's ["Getting the code"](https://github.com/mozilla/DeepSpeech/blob/v0.1.1/README.md#getting-the-code) chapter
-- `cd` into the directory you've just cloned the project into
-- run the following commands:
-
-        $ git checkout v0.1.1
-        $ sudo mkdir /usr/local/include/DeepSpeech
-        $ sudo cp native_client/*.h /usr/local/include/DeepSpeech
-        $ sudo cp native_client/build/*.so /usr/local/lib
-        
-- make sure `/usr/local/lib` is in your `LD_LIBRARY_PATH` environment variable
+- fetch an uptodate `native_client.tar.xz` matching your system, from DeepSpeech's ["releases"](https://github.com/mozilla/DeepSpeech/releases/tag/v0.4.0-alpha.3)
+- download `deepspeech.h` from https://github.com/mozilla/DeepSpeech/raw/v0.4.0-alpha.3/native_client/deepspeech.h
```

Comment:
Is it safe to use the `v0.4.0` release instead of `v0.4.0-alpha.3`?

Reply:
I've used v0.4.0-alpha.3 there to ensure you could fetch and test, v0.4.0 is not yet a tag. Once it is, it should be used instead of v0.4.0-alpha.3. It should be a matter of days now.

Reply:
OK, I'd rather you update the tag now in the PR so that we don't forget if you don't mind. I'm afraid we'll forget to change it afterwards...

Reply:
@asticode Should be `v0.4.0` now :)